functions/Get-DbaDatabaseFile.ps1

```diff
@@ -82,7 +82,7 @@ function Get-DbaDatabaseFile {
             mf.physical_name as PhysicalName,
             df.state_desc as State,
             df.max_size as MaxSize,
-            df.growth as Growth,
+            case mf.is_percent_growth when 1 then df.growth else df.Growth*8 end as Growth,,
```

Comment:
Check the end of the line...that is why the test fails.

AS well around line 172 we need to add a debug statement to output that query. Something like this is what we have been using:

`Write-Message -Level Debug -Message "SQL Statement: $query"`